docs/dev/setup/bamboo-bitbucket-jira.rst

```diff
@@ -75,19 +75,31 @@ under ``localhost:7990``.
 
 **Get evaluation licenses for Atlassian products:** `Atlassian Licenses <https://my.atlassian.com/license/evaluation>`__
 
-1. Create an admin user with the same credentials in all 3 applications.
-   Create a sample project in Jira. Also, you can select the
-   evaluation/internal/test/dev setups if you are asked. Select a
-   ``Bitbucket (Server)`` license if asked. Do not connect Bitbucket
-   with Jira yet.
-
-2. | Execute the shell script ``atlassian-setup.sh`` in the
-     ``src/main/docker`` directory (e.g. with
-     ``src/main/docker/./atlassian-setup.sh``). This script creates
-     groups, users ([STRIKEOUT:and adds them to the created groups] NOT
-     YET) and disabled application links between the 3 applications
-
-3. Enable the created `application
+1. Get licenses for Bamboo, Bitbucket and Jira Service Management.
+
+   - Bamboo: Select Bamboo (Server) and ``not installed yet``
+   - Bitbucket: Select Bitbucket (Data Center) and ``not installed yet``
+   - Jira: Select Jira Service Management (Data Center) and ``not installed yet``
+
+2. Provide the just created license key during the setup and create an admin user with the same credentials in all 3 applications.
+   Also, you can select the evaluation/internal/test/dev setups if you are asked.
+   Follow the additional steps for Jira and Bitbucket.
+
+   - Jira:
+    - On startup select ``I'll set it up myself``
+    - Select Build In Database Connection
+    - Create a sample project
+
+   - Bitbucket: Do not connect Bitbucket with Jira yet
+
```

Comment:
Maybe add a step that makes sure that `xdg-utils` are installed. We found an issue that occured when the `xdg-open` command was not found.
```suggestion
3. Make sure that `xdg-utils` is installed before running the script. `xdg-utils <https://www.howtoinstall.me/ubuntu/18-04/xdg-utils/>`
```